crates/compiler/test_syntax/tests/test_snapshots.rs

```diff
@@ -730,6 +765,13 @@ mod test_snapshots {
                 true
             }
 
+            "parens_newlines_before_as" => {
+                // Found unbound type variables {66}
+                //  in type alias `Test.IdentId(0)` [] [] : <66>
+                // Location: crates/compiler/can/src/scope.rs:541:13
```

Comment:
What's going on here?

Reply:
This is the panic in canonicalization that these two tests run into. I'll get to this eventually, but getting fuzz clean is higher on my list.

Or, that could be an interesting first issue for an enterprising individual!